File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] output = new int[2]; // Initialize array

        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) { // Fix loop condition
                if (nums[i] + nums[j] == target) {
                    output[0] = i;
                    output[1] = j;
                    return output; // Return immediately after finding a solution
                }
            }
        }

        return new int[]{}; // Return an empty array if no solution is found
    }
}

File: Data Structures & Algorithms/binary-search/submission-0.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        low = 0
        high = len(nums)-1
        mid = 0

        while low <= high:
            mid = (high + low)
            if nums[mid] == target:
                return mid
            elif target > nums[mid]:
                low = mid +1
            else:
                high = mid-1

        return -1
            
        

        



